tests/sla_tests.js

```diff
@@ -125,11 +125,14 @@ var getMockChannel = function() {
     this.id = channelId.toString();
     channelId += 1;
     this.originate = function(input, cb) {
-      this.dialed = true;
+      this['dialed'] = 'yes';
```

Comment:
Looks like you overwrote changes made by Chris that have already been merged to master.